src/components/order-service/order-service.controller.ts

```diff
@@ -202,19 +202,10 @@ export class OrderServiceController implements angular.IController {
       let newProjName = this.ctrl.selectedProject.metadata.name;
       let newProjDisplayName = this.ctrl.selectedProject.metadata.annotations['new-display-name'];
       let newProjDesc = this.$filter('description')(this.ctrl.selectedProject);
-      let projReqObj: any = {
-        apiVersion: "v1",
-        kind: "ProjectRequest",
-        metadata: {
-          name: newProjName
-        },
-        displayName: newProjDisplayName,
-        description: newProjDesc
-      };
-      this.DataService
-        .create('projectrequests', null, projReqObj, this.$scope)
-        .then( (data: any) => {
-          this.ctrl.selectedProject = data;
+      this.ProjectsService
+        .create(newProjName, newProjDisplayName, newProjDesc)
+        .then( (project: any) => {
+          this.ctrl.selectedProject = project;
           this.ctrl.projectDisplayName = this.$filter('displayName')(this.ctrl.selectedProject);
```

Comment:
Not new, but let's pass `project` here instead of `ctrl.selectedProject`. The project request template could potentially change the display name, so better to use what we get back.